docs/usage/8.md

```diff
@@ -0,0 +1,298 @@
+export new_noobaa_sha=$(oc image mirror --dry-run --insecure=true ${remote_registry}/${remote_registry_ns}/${noobaa_image}@sha256:${noobaa_sha} ${exposed_registry}/${local_registry_ns}/mcg-core-rhel8:v4.2.0 2>&1 | grep -A1 manifests | grep sha256 | awk -F'[: ]' '{ print $8 }')
+sed -i "s/${roperator_sha}/${new_roperator_sha}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+sed -i "s/${moperator_sha}/${new_moperator_sha}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+sed -i "s/${ooperator_sha}/${new_ooperator_sha}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+sed -i "s/${noobaa_sha}/${new_noobaa_sha}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+```
+
+### Update the OCS operator source and namespaces for dev/stage
+host unknown errors seem to stop the image mirror policy from working? The same with namespace and permissions to it? These claims need verification.
+```
+if [[ "registry.redhat.io" != "${source_registry}" ]]; then
+  sed -i "s/${ocs_source_registry}/${source_registry}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+  sed -i "s/${ocs_source_registry_ns}/${source_registry_ns}/g" manifests/ocs-operator/4.2.0/ocs-operator.v4.2.0.clusterserviceversion.yaml
+fi
+```
+
+## Step 4-5: Skip
+
+## Step 6a: Write the Dockerfile for the custom registry
+```
+cat << EOF > Dockerfile
+FROM registry.redhat.io/openshift4/ose-operator-registry AS builder
+
+COPY manifests manifests
+
+RUN /bin/initializer -o ./bundles.db
+
+FROM registry.redhat.io/rhel7
+
+COPY --from=builder /registry/bundles.db /bundles.db
+COPY --from=builder /bin/registry-server /registry-server
+COPY --from=builder /bin/grpc_health_probe /bin/grpc_health_probe
+
+EXPOSE 50051
+
+ENTRYPOINT ["/registry-server"]
+
+CMD ["--database", "bundles.db"]> Dockerfile
+EOF
+```
+
+## Step 6b: Build the container 
+```
+docker build -f Dockerfile -t ${local_app_registry}/${local_app_ns}/custom-registry .
+```
+
+## Step 7: Push the container
+```
+docker push ${local_app_registry}/${local_app_ns}/custom-registry
+```
+
+Ensure the repository is public from the quay uI
+
+## Step 8a: Create openshift-migration Namespace and Catalog Source
+```
+oc create namespace openshift-migration
+oc project openshift-migration
+
+cat << EOF > cam-catalog.yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: cam-catalog
+  namespace: openshift-migration
+spec:
+  displayName: CAM Catalog
+  sourceType: grpc
+  image: ${local_app_registry}/${local_app_ns}/custom-registry:latest
+EOF
+```
+
+## Step 8b: Install the Catalog Source
+```
+oc create -f cam-catalog.yaml
+```
+
+## Step 9a: Skip
```

Comment:
Should we explain why this step is skippable in a sentence?

Reply:
There's a lot of places where they just want you to find and look at things. For instance, steps 2-3b it's all to get the tarball... using jq I just turmed it into a one liner... here I think they say something like find the list of images again... there's nothing to do if you copy/paste the commands.